components/homme/test_execs/share_kokkos_ut/CMakeLists.txt

```diff
@@ -9,8 +9,8 @@ SET(UTILS_TIMING_DIR ${HOMME_SOURCE_DIR}/utils/cime/CIME/non_py/src/timing)
 # Note: need CUDA_BUILD and HOMMEXX_BFB_TESTING here, since the share
 #       unit tests do not include a config.h file
 SET (COMMON_DEFINITIONS NP=4 NC=4)
-IF (CUDA_BUILD)
-  SET(COMMON_DEFINITIONS ${COMMON_DEFINITIONS} CUDA_BUILD)
+IF (CUDA_BUILD OR HIP_BUILD)
```

Comment:
Could simply do `IF (HOMMEXX_ENABLE_GPU)`, like it's done elsewhere, so it's ready if/when we add SYCL.

Reply:
Thanks -- this one was discussed before, there is CUDA_BUILD used elsewhere in homme cmake. It can be cleaned up later.